### PR TITLE
Update specs to 16.3.0

### PR DESCRIPTION
**Pull request by kddejong**
*Issue #, if available:*

*Description of changes:*
- Update specs to 16.3.0

```
scripts/update_specs_from_pricing.py # requires Boto3 and Credentials
scripts/update_specs_services_from_ssm.py # requires Boto3 and Credentials
cfn-lint --update-specs # https://github.com/aws-cloudformation/cfn-python-lint/pull/1383\#issuecomment-629891506
cfn-lint --update-iam-policies
cfn-lint --update-documentation
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json [hunk 1]
```diff
@@ -122,9 +122,18 @@
...
+  },
+  {
+    "op": "add",
     "path": "/ResourceTypes/AWS::IoT::ProvisioningTemplate/Properties/Tags/ItemType",
     "value": "Tag"
   },
+  {
+    "op": "remove",
+    "path": "/PropertyTypes/AWS::IoT::ProvisioningTemplate.Tags"
+  },
```
Comment (PatMyron) [inline src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json:136]: just a comment to call this out as manual patching

**Comment by kddejong** [comment]
Manually changed the already existent patching for `AWS::IoT::ProvisioningTemplate.Tags`.

File: src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json (hunk 1 above)
Comment (kddejong) [reply src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json:136]: Agreed.  We were already patching it but looks like they made some changes that are still not correct.

File: src/cfnlint/data/ExtendedSpecs/cn-north-1/05_pricing_property_values.json [hunk 2]
```diff
@@ -1 +1,61 @@
-[]
```
Comment (PatMyron) [inline src/cfnlint/data/ExtendedSpecs/cn-north-1/05_pricing_property_values.json:1]: AWS China pricing information https://github.com/aws-cloudformation/cfn-python-lint/pull/1535

File: src/cfnlint/data/ExtendedSpecs/cn-northwest-1/05_pricing_property_values.json [hunk 3]
```diff
@@ -1 +1,56 @@
-[]
```
Comment (PatMyron) [inline src/cfnlint/data/ExtendedSpecs/cn-northwest-1/05_pricing_property_values.json:1]: instance type pricing information finally working for AWS China regions: https://github.com/aws-cloudformation/cfn-python-lint/pull/1535

**Pull request merged**